domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -151,6 +156,9 @@ class ProfileManagementController @Inject constructor(
         .setAllowDownloadAccess(allowDownloadAccess)
         .setId(ProfileId.newBuilder().setInternalId(nextProfileId))
         .setDateCreatedTimestampMs(Date().time).setIsAdmin(isAdmin)
+        .setStoryTextSize(storyTextSize!!)
```

Comment:
Why allow a null story text size if it's assumed to not be null here?

Reply:
Its resolved.